crates/iota-json-rpc-types/src/iota_extended.rs

```diff
@@ -223,3 +224,19 @@ pub struct AddressMetrics {
     /// The count of daily unique sender addresses.
     pub daily_active_addresses: u64,
 }
+
+/// Provides a summary of the circulating IOTA supply.
+#[serde_as]
+#[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct IotaCirculatingSupplySummary {
+    /// Circulating supply in IOTA at the given timestamp.
+    pub value: f64,
+    /// Percentage of total supply that is currently circulating (range: 0.0 to
+    /// 1.0).
+    pub circulating_supply_percentage: f64,
+    /// Timestamp (UTC) when the circulating supply was calculated.
+    #[serde(with = "chrono::serde::ts_seconds")]
+    #[schemars(with = "i64")]
+    pub timestamp: DateTime<Utc>,
```

Comment:
Do requests work if the indexer is not synced up to the latest checkpoint? If not, then this could end up being wrong
Don't we usually return the checkpoint instead? Or maybe both, timestamp and checkpoint